e2e/cypress/integration/integrations/slash_commands_spec.js

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @integrations
+
+describe('Integrations', () => {
+    let user1;
+    let user2;
+    let team1;
+    let team2;
+    let testChannelUrl1;
+    let testChannelUrl2;
+    const commandTrigger = 'test-ephemeral';
+    const commandURL = 'http://hidden-peak-21733.herokuapp.com/test_ephemeral';
+    const timestamp = Date.now();
+
+    before(() => {
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                EnableLinkPreviews: true,
+                EnableOnlyAdminIntegrations: false,
+            },
+        });
+
+        cy.apiInitSetup().then(({team, user}) => {
+            user1 = user;
+            team1 = team;
+            testChannelUrl1 = `/${team.name}/channels/town-square`;
+
+            cy.apiCreateUser().then(({user: otherUser}) => {
+                user2 = otherUser;
+
+                cy.apiAddUserToTeam(team.id, user2.id);
+            });
+
+            cy.apiCreateTeam(`test-team-${timestamp}`, `test-team-${timestamp}`).then(({anotherTeam}) => {
```

Comment:
Should this be?
```suggestion
            cy.apiCreateTeam(`test-team-${timestamp}`, `test-team-${timestamp}`).then(({team: anotherTeam}) => {
```

Reply:
These lines should solve this. Maybe I forgot something more in the configuration of the server?

https://github.com/mattermost/mattermost-webapp/blob/1b1c83883b6b77cb39e9501c1e8644f61e407e2c/e2e/cypress/integration/integrations/slash_commands_spec.js#L25-L30